### ISSUE TITLE
Panel clock not updating unless highlighted by mouse

### ISSUE THREAD

**Issue by jrvanwhy** [lxde/lxqt#883]
The time indicated on my clock does not update unless I mouse over the clock applet, so the displayed time is usually out of date.

Specifically, the clock time updates when I move the mouse to the clock, when I move away from the clock, and when I click on the clock. Hovering or moving the mouse within the bounding box of the clock does not cause it to update.

I am running Arch Linux on x86_64 and have just updated from LXQt 0.9 to LXQt 0.10.0.

My panel is configured to run vertically along the right side of my screen and I have the "Autorotate when the panel is vertical" so the clock is rotated to match my vertical panel.

Let me know if you would like me to provide any more information. If there are instructions on how to build and run the clock plugin from source, please let me know -- I would start debugging it, but I do not know how to compile panel plugins.

**Comment by jrvanwhy** [comment]
Okay, I have it building and have done a bit of debugging.

I have confirmed that [line 163 of lxqtclock.cpp](https://github.com/lxde/lxqt-panel/blob/master/plugin-clock/lxqtclock.cpp#L163) is correctly passing an updated time to QLabel::setText() each second, so the timer is definitely working correctly.

I've also confirmed that calling QLabel::update() immediately after QLabel::setText() does not solve the problem.

Perhaps Qt isn't getting the chance to process the label update? I'm out of time for debugging right now, but perhaps that helps someone else solve this problem.
